benchmark/preconditioner/preconditioner.cpp

```diff
@@ -183,6 +183,7 @@ struct PreconditionerBenchmark : Benchmark<preconditioner_benchmark_state> {
     {
         preconditioner_benchmark_state state;
         auto data = Generator::generate_matrix_data(test_case);
+        reorder(data, test_case);
```

Comment:
Do these do anything in practice, or is it only to note that it is potentially reordered? Since the permutation isn't stored.

Reply:
the functions modify `data` and add an entry to `test_case`